spec/models/location_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+describe Location, type: :model do
+  describe 'relationships' do
+    it { should have_many :bookstores }
+  end
+
+  describe 'methods' do
+    describe '#sort_by_recently_created' do
+      it 'sorts the locations by most recently created first and displays when created' do
+        location_1 = Location.create({
+          name: "Philadelphia",
+          population: 15000000,
+          urban: true})
+    sleep(2)
+        location_2 = Location.create({
+          name: "New York City",
+          population: 35000000,
+          urban: true})
+
+        expect(Location.sort_by_recently_created.first).to eq (location_2)
+        expect(Location.sort_by_recently_created.last).to eq (location_1)
+      end
+    end
+
+    describe '#bookstore_count' do
+      it 'counts the number of bookstores linked to that location' do
+        location_1 = Location.create({
```

Comment:
With `.create` here, does this need to be `.create!` similar  to how it is with `bookstore_1` and `bookstore_2`?